modules/update-service-mirror-release.adoc

```diff
@@ -0,0 +1,211 @@
+[id="update-service-mirror-release_{context}"]
+= Mirroring the {product-title} image repository
+
+The OpenShift Update Service requires a locally accessible registry containing update release payloads. 
+
+[IMPORTANT]
+====
+To avoid excessive memory usage by the OpenShift Update Service application the release images should be mirrored to a separate repository. To accomplish this you must follow the instructions below.
+====
+
+.Prerequisites
+
+* Review and complete the steps from xref:../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[Creating a mirror registry for installation in a restricted network] up to, but not including, section *Mirroring the {product-title} image repository*.
+* Your mirror host has access to the Internet.
+* You configured a mirror registry to use in your restricted network and
+can access the certificate and credentials that you configured.
+ifndef::openshift-origin[]
+* You downloaded the pull secret from the
+link:https://cloud.redhat.com/openshift/install/pull-secret[Pull Secret] page on the {cloud-redhat-com} site and modified it to include authentication to your mirror repository.
+endif::[]
+ifdef::openshift-origin[]
+* You have created a pull secret for your mirror repository.
+endif::[]
+* If you use self-signed certificates that do not set a Subject Alternative Name, you must precede the `oc` commands in this procedure with `GODEBUG=x509ignoreCN=0`. If you do not set this variable, the `oc` commands will fail with the following error:
++
+[source,terminal]
+----
+x509: certificate relies on legacy Common Name field, use SANs or temporarily enable Common Name matching with GODEBUG=x509ignoreCN=0
```

Comment:
Looks like these actions have been covered in **Creating a mirror registry for installation in a restricted network**. So I feel like there's no need to repeat. WDYT? 
To retrieve the available updates, the current release running on the cluster should be accessible from the mirror registry. So I would suggest to add a prerequisite like: The current release running on the cluster has been mirrored on the mirror registry.

Reply:
Per the prerequisite `Review and complete the steps from Creating a mirror registry for installation in a restricted network up to, but not including, section Mirroring the OpenShift Container Platform image repository`m user will do each of those sections down to, but not including, `Mirroring the OpenShift Container Platform image repository` from the general instructions and then do this section `Mirroring the OpenShift Container Platform image repository` here which completely replaces the other.

Reply:
I got your point. It's okay for me.
One more concern, it's saying **Your mirror host has access to the Internet**. However, procedure 3 is saying **If your mirror host does not have Internet access, take the following actions**. They are two apparently contradictory statements.

Reply:
Yeah, you're right. That's how the original section is written as well but I agree, the host doesn't have to be connected to the internet so I removed that prereq.